common/templates/layouts/collection.njk

```diff
@@ -7,11 +7,10 @@
 {% endblock %}
 
 {% if period === 'week' %}
-  {% set datetime = dateRange | formatISOWeek %}
-  {% set displayDate = dateRange | formatDateRangeAsRelativeWeek %}
+  {% set displayDate = dateRange | formatDateRangeWithRelativeWeek %}
 {% else %}
   {% set datetime = dateRange | first %}
-  {% set displayDate = datetime | formatDateAsRelativeDay %}
+  {% set displayDate = datetime | formatDateWithRelativeDay %}
 {% endif %}
 
 {% block pageTitle %}
```

Comment:
```
{{ t("collections::page_title_with_date", {
    context: context,
    date: displayDate | lower
  }) }}
```
Can we remove the `| lower` in the title? Now we're not saying "Outgoing moves for today", I don't think we need it. Now we get the somewhat weird "Outgoing moves for wednesday 19 aug 2020 (today)"